ensure-is-media.js

```diff
@@ -9,16 +9,57 @@ let makeUseIsMedia = (
 // improving the algorithms inside, etc, see this:
 // https://github.com/viewstools/morph/blob/master/ensure-is-media.js
 
-import { useMedia } from 'use-media';
+import { useState, useEffect } from 'react'
+
+const MOCK_MEDIA_QUERY_LIST = {
+  media: '',
+  matches: false,
+  onchange: () => {},
+  addListener: () => {},
+  removeListener: () => {},
+  addEventListener: () => {},
+  removeEventListener: () => {},
+  dispatchEvent: _ => true, // dispatchEvent: (_: Event) => true
+}
+
+let useMedia = (
+  query,
+  defaultState = window.matchMedia(query).matches
+) => {
+  const [state, setState] = useState(defaultState)
+
+  useEffect(() => {
+    let mounted = true
```

Comment:
let's rename `mounted` to `cancel` to match how we define that in the rest of the apps